chainer/links/connection/convolution_nd.py

```diff
@@ -57,11 +57,15 @@ class ConvolutionND(link.Link):
 
     """
 
-    def __init__(self, ndim, in_channels, out_channels, ksize, stride=1, pad=0,
-                 nobias=False, initialW=None, initial_bias=None,
+    def __init__(self, ndim, in_channels, out_channels, ksize=None, stride=1,
+                 pad=0, obias=False, initialW=None, initial_bias=None,
```

Comment:
Ops, `obias` a careless?

Reply:
@hvy Thanks, I fixed this point in the latest commit already. Please check it again :)